okapi-common/src/main/java/org/folio/okapi/common/ModuleId.java

```diff
@@ -1,34 +1,101 @@
 package org.folio.okapi.common;
 
 import java.util.Collection;
+import java.util.List;
 
 public class ModuleId implements Comparable<ModuleId> {
 
+  private static final String MODULE_QUOTE = "ModuleID '";
```

Comment:
This is odd. How about `IllegalModuleIdException(String msg)` and `IllegalModuleIdException(String moduleId, String msg)`?